### ISSUE TITLE
Update TeamType model

### ISSUE THREAD

**Issue by jcdesimp** [andymeneely/dev-fortress-server#69]
TeamType model should change one field and add another
* `objective - Integer`
* change `initial_mature - bool` to `mature_at - Integer`

**Comment by ShaydeNofziger** [comment]
Based on existing documentation, these are the teams and their objective/mature at values:

Team | Objective (DevCaps) | Mature At (Round Number)
-- | -- | -- 
iOS | 30 | 3
Android | 30 | 3
Web | 28 | 1*
API | 20 | 2
Cloud Services | 24 | 1*
Automated Warehouse | 30 | 2
Drone Delivery | 30 | 3
Streaming App | 40 | 0^

_* Indicates the team **begins** the game as Mature._
_^ Indicates the team will **never** be Mature._

**Comment by ShaydeNofziger** [comment]
@andymeneely Should these fields be modifiable by the Storyteller on the `Team` model, much like the other _initial..._ attributes?

**Comment by andymeneely** [comment]
I don't _need_ it per se. Once a game is started, I won't be changing objectives or maturity. I would balance it other ways.

**Comment by ShaydeNofziger** [comment]
@andymeneely Sorry, I don't think I communicated my question clearly enough...

When the Storyteller is configuring a **new** game, and is configuring the Teams, should they be able to modify those values (prior to sending the out the Team link codes)?

It's a bit nit-picky, but determines whether the `Team` model requires the field as well.

**Comment by ShaydeNofziger** [comment]
Although the same balance could be achieved by event damage modifiers and event selections, I suppose, which is what your comment was getting at.

**Comment by andymeneely** [comment]
Yeah. At that point I'd probably be re-seeding the database anyway so I still probably don't need it.
